### ISSUE TITLE
Using browser's search function (Ctrl+F) with large data of grid gives incorrect total number

### ISSUE THREAD

**Issue by elfwine** [vitmalina/w2ui#2545]
**Short description**
When Grid have a large record data (1000+) and user used the browser's search function (Ctrl+F) then search for certain keyword, the results is only applicable to the visible grid data. when the user keeps on pressing arrow-down or arrow-up button, results' total number changes.
Is there anyway to integrate or override browser's search function to grid's search function?

**What is current behavior**
When using browser's search function(Ctrl+F), total number of results is inconsistent.

**What is desired behavior**

**Link to jsfidle/codepan with sample code**
https://w2ui.com/web/demos/#/combo/4

**Steps to reproduce or sample** 
Press Ctrl+F, then try to find for 'Sidorov' and keeps pressing arrow-down ( v )

**Comment by vitmalina** [comment]
I am not sure it is possible as browsers' ctrl+F will search what exists in the browser. You can override browser's ctrl+F and submit grid search.

Or if you have other suggestions, let me know

**Comment by elfwine** [comment]
Yes, I ended up overriding browser's ctrl+F (mimicking it's ui and function).
To get the total ctrl+f search count, return the count number of keyword found in grid records.
For keyword's highlight, I had to watch grid's scroll event and apply the highlight marker on displayed grid.
I couldn't use the grid's search and it's marker as it will reduce the grid records display.
Also, I suggested/showed grid's search function to use but it was rejected as users use ctrl+f more often.
